lms/djangoapps/program_enrollments/management/commands/send_program_course_nudge_email.py

```diff
@@ -124,17 +125,28 @@ def get_program(self, programs, program_progress):
             if program['uuid'] == program_progress['uuid']:
                 return program
 
-    def emit_event(self, user, program, suggested_course_run, completed_course_run):
+    def emit_event(self, user, program, suggested_course_run, suggested_course, completed_course_run):
         """
          Emit the Segment event which will be used by Braze to send the email
         """
+        learner_data = get_enterprise_learner_data_from_db(user)
+        enterprise_customer = learner_data[0]['enterprise_customer'] if learner_data else None
+        if enterprise_customer and enterprise_customer['enable_learner_portal']:
+            # If user is an enterprise learner then we want to redirect him to B2B course landing on learner portal.
```

Comment:
Please remove `him`. What about `an enterprise learner will be redirected to B2B course landing on learner portal`?